src/phantom.js

```diff
@@ -107,6 +107,15 @@ export default class Phantom {
      * Returns a value in the global space of phantom process
      * @returns {Promise}
      */
```

Comment:
Maybe a better option would be to do 

```js
this.logger = logger
``` 
in the constructor so that others can do whatever they want.